packages/core/contracts/umip-helpers/VotingUpgraderV2.sol

```diff
@@ -91,6 +97,15 @@ contract VotingUpgraderV2 {
         multiroleContracts = _multiroleContracts;
     }
 
+    /**
+     * @notice Checks if the caller is the upgrader.
+     * @dev This is used as the first transaction in the upgrade process to block any other transactions from being
+     * executed if the upgrade is not initiated by the upgrader.
+     */
+    function canRun() public view {
```

Comment:
should this be external if not called from this contract

Reply:
does not really make a functional difference I don't think but you are right